### PR TITLE
pagination doesnt update while searching fix

### LINKED ISSUE #560: Pagination per page not update while we search (All pages search option Receive/Addressbook)
How to Reproduce:
1)When you are on address book page, Initially you are on first page and you do the search and get the results correctly.
2)But when you move to second page and try to search result for the same page or previous page result it will not give you any result.

screen shot 2017-12-29 at 4 03 18 pm
screen shot 2017-12-29 at 4 03 29 pm

### PR DESCRIPTION
**Pull request by vikas-cis**
#560

### CODE REVIEW

File: src/app/wallet/wallet/receive/receive.component.ts [hunk 1]
```diff
@@ -343,4 +343,11 @@ export class ReceiveComponent implements OnInit {
       error => this.log.er('error'));
     }
   }
+
+  public resetPagination(): void {
```
Comment (anandsinghparihar) [inline src/app/wallet/wallet/receive/receive.component.ts]: why we don't create a sing function which reset paginations?

File: src/app/wallet/wallet/shared/address-table/address-table.component.ts [hunk 2]
```diff
@@ -184,5 +185,12 @@ export class AddressTableComponent implements OnInit {
     dialogRef.componentInstance.formData.address = address;
     dialogRef.componentInstance.type = 'verify';
   }
+
+  public resetPagination(): void {
```
Comment (anandsinghparihar) [inline src/app/wallet/wallet/shared/address-table/address-table.component.ts]: a single function will refactor same code. 
so please create a sing function which reset paginations?

**Comment by pciavald** [comment]
@vikas-cis do not forget to lint next time 🔪 😄

**Comment by vikas-cis** [comment]
@pciavald Thanks Pierre, Will take care 😄

**Pull request merged**